WootricSDK/WootricSDK/WTRSettings.h

```diff
@@ -65,6 +70,12 @@
 - (NSString *)socialShareQuestionText;
 - (NSString *)socialShareDeclineText;
 
+- (UIColor *)sendButtonBackgroundColor;
+- (UIColor *)noThanksColor;
+- (UIColor *)sliderColor;
+- (UIColor *)thankYouButtonBackgroundColor;
+- (UIColor *)socialSharingColor;
+
```

Comment:
You're right, I do need -thankYouButtonBackgroundColor though. That property is not declared in this class.